pymarl/config/envs/struct_c_10.yaml

```diff
@@ -0,0 +1,23 @@
+env: struct
+
+env_args:
+  n_comp: 10
+  discount_reward: 0.95
+  k_comp: 9
+  state_obs: True
+  state_d_rate: True
+  state_alphas: True
+  obs_d_rate: False
+  obs_multiple: False
+  obs_all_d_rate: False
+  obs_alphas: False
```

Comment:
Do we add obs_alphas for the correlated env?

Reply:
In my previous experiments, I see that adding alphas improves a lot the results.

Reply:
Done in [2aa7a1f](https://github.com/moratodpg/marl_struct/pull/16/commits/2aa7a1f55b0e3d61373594f5f4c16ffb2aec85d6)